src/screens/Calendar.js

```diff
@@ -401,6 +401,15 @@ const CalendarPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="calendar.large.title">
```

Comment:
Can we add a note to `calendar.heading.level` that `calendar[size].title` should be used instead?

Reply:
aw yes good point